packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -370,3 +380,9 @@ ProductTile.propTypes = {
 }
 
 export default ProductTile
+
+const shouldShowPromoCallout = (productWithFilteredVariants) => {
```

Comment:
minor: make it a helper function` hasPromotions(product)` in product-utils?

Reply:
I don't want to expose this particular function more globally. For example, due to the inconsistent API (productSearch vs getProduct), I should not be calling it on the Product Details page.  So I'd rather constrain this function within ProductTile only.